ibm_catalog.json

```diff
@@ -18,21 +18,34 @@
         "terraform",
         "solution",
         "cloudant dedicated",
-        "cloud dedicated"
+        "database",
+        "nosql"
       ],
-      "short_description": "Creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated environment.",
-      "long_description": "This architecture creates an IBM Cloudant Dedicated instance in IBM Cloud Dedicated environment and supports provisioning the following resources:\n- A resource group, if one is not passed in.\n- An IBM Cloudant Dedicated instance on IBM Cloud Dedicated\n- A IBM Cloudant database\n",
-      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cloudant/blob/main/solutions/dedicated/README.md",
-      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cloudant/images/cloudant.svg",
+      "short_description": "Creates and configures an IBM Cloudant instance.",
+      "long_description": "This architecture supports creating and configuring an IBM Cloudant instance and database.",
+      "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-cloudant/blob/main/README.md",
+      "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-cloudant/main/images/cloudant.svg",
       "provider_name": "IBM",
       "features": [
         {
           "title": "Creates an IBM Cloudant Dedicated instance",
-          "description": "Creates an IBM Cloudant Dedicated instance on IBM Cloud Dedicated environment."
+          "description": "Creates an IBM Cloudant Dedicated instance."
```

Comment:
remove `Dedicated` from here - all flavours may not be for dedicated in the future